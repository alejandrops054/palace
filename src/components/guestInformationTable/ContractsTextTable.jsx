/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos 
*/

import React from "react";
import Table from "rc-table";
import { Input, Select, Button } from "../index";

const columns = [
  {
    title: "Number",
    dataIndex: "number",
    key: "number",
    width: 200,
  },
  {
    title: "Text",
    dataIndex: "text",
    key: "text",
    width: 200,
  },
  {
    title: "Status",
    dataIndex: "status",
    key: "status",
    width: 200,
  },
  
];

const data = [
  {
    number: '11',
    text: 'ZAPATERIA 3 HERMANOS',
    status: <Select option="Alta " col={9} />
   
  },
];

export default function ContratsTextTable() {
  return (
    <form className="mb-6">
      <div className="card">
        <br />
        <h3>Member Information</h3>
        <br />
        <div className="row">
                <div className="col-md-4">
                <Input type='text' label='Company' placeholder='5' col={12}/>
                </div>
                <div className="col-md-4">
                <Input type='text' label='Application' placeholder='60007770' col={12}/>
                </div>
                <div className="col-md-4">
                <Input type='text' label='Member Name' placeholder='WILLIAM TORRES FLOTA' col={12}/>
                </div>
            </div>
            <div className="row">
                <div className="col-md-4">
                <Input type='text' label='Office' placeholder='11' col={12}/>
                </div>
                <div className="col-md-4">
                <Input type='text' label='PLAYACAR PALACE' placeholder='' col={12}/>
                </div>
                <div className="col-md-4">
                <Input type='text' label='Project' placeholder='PALACE ELITE' col={12}/>
                </div>
            </div>
        <br />
        <br />
        <div>
          <Table
            className="table"
            columns={columns}
            data={data}
          />
        </div>
        <br/>
         <div className="row">
         <div className="col-md-3">
            <Button title="Log" col="5" typebootstrap="secondary" />
            </div>
            <div className="col-md-3">
            <Button title="Add" col="5" typebootstrap="secondary" />
            </div>
            <div className="col-md-3">
            <Button title="Properties" col="8" typebootstrap="secondary" />
            </div>
            <div className="col-md-3">
            <Button title="Close" col="6" typebootstrap="primary" />

            </div>
          </div>
        </div>
     </form>
  );
}
