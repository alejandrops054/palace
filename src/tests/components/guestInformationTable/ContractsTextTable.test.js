/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import ContractsTextTable from '../../../components/guestInformationTable/ContractsTextTable'

describe('ContractsTextTable component', () => {
  test('debe mostrar snapshot', () => {

    const wrapper = shallow(<ContractsTextTable />)
    expect(wrapper).toMatchSnapshot();
  })

})