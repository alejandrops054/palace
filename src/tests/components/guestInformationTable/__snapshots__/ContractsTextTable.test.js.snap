// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContractsTextTable component debe mostrar snapshot 1`] = `
<form
  className="mb-6"
>
  <div
    className="card"
  >
    <br />
    <h3>
      Member Information
    </h3>
    <br />
    <div
      className="row"
    >
      <div
        className="col-md-4"
      >
        <Input
          col={12}
          label="Company"
          placeholder="5"
          type="text"
        />
      </div>
      <div
        className="col-md-4"
      >
        <Input
          col={12}
          label="Application"
          placeholder="60007770"
          type="text"
        />
      </div>
      <div
        className="col-md-4"
      >
        <Input
          col={12}
          label="Member Name"
          placeholder="WILLIAM TORRES FLOTA"
          type="text"
        />
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-4"
      >
        <Input
          col={12}
          label="Office"
          placeholder="11"
          type="text"
        />
      </div>
      <div
        className="col-md-4"
      >
        <Input
          col={12}
          label="PLAYACAR PALACE"
          placeholder=""
          type="text"
        />
      </div>
      <div
        className="col-md-4"
      >
        <Input
          col={12}
          label="Project"
          placeholder="PALACE ELITE"
          type="text"
        />
      </div>
    </div>
    <br />
    <br />
    <div>
      <Table
        className="table"
        columns={
          Array [
            Object {
              "dataIndex": "number",
              "key": "number",
              "title": "Number",
              "width": 200,
            },
            Object {
              "dataIndex": "text",
              "key": "text",
              "title": "Text",
              "width": 200,
            },
            Object {
              "dataIndex": "status",
              "key": "status",
              "title": "Status",
              "width": 200,
            },
          ]
        }
        data={
          Array [
            Object {
              "number": "11",
              "status": <Select
                col={9}
                option="Alta "
              />,
              "text": "ZAPATERIA 3 HERMANOS",
            },
          ]
        }
        emptyText={[Function]}
        prefixCls="rc-table"
        rowKey="key"
      />
    </div>
    <br />
    <div
      className="row"
    >
      <div
        className="col-md-3"
      >
        <Button
          col="5"
          title="Log"
          typebootstrap="secondary"
        />
      </div>
      <div
        className="col-md-3"
      >
        <Button
          col="5"
          title="Add"
          typebootstrap="secondary"
        />
      </div>
      <div
        className="col-md-3"
      >
        <Button
          col="8"
          title="Properties"
          typebootstrap="secondary"
        />
      </div>
      <div
        className="col-md-3"
      >
        <Button
          col="6"
          title="Close"
          typebootstrap="primary"
        />
      </div>
    </div>
  </div>
</form>
`;
